### PR TITLE
typo: Double word "use"

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Cdn/Cdn/Endpoint/NewAzCdnDeliveryRuleAction.cs [hunk 1]
```diff
@@ -71,7 +71,7 @@ public class NewAzCdnDeliveryRuleAction : AzureCdnCmdletBase
         [ValidateNotNullOrEmpty]
         public string CustomPath { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Host to redirect. Leave empty to use use the incoming host as the destination host.", ParameterSetName = UrlRedirectActionParameterSet)]
+        [Parameter(Mandatory = false, HelpMessage = "Host to redirect. Leave empty to use the incoming host as the destination host.", ParameterSetName = UrlRedirectActionParameterSet)]
```
Comment (markcowl) [inline src/Cdn/Cdn/Endpoint/NewAzCdnDeliveryRuleAction.cs]: Requires an entry in the Cdn changelog

File: src/FrontDoor/FrontDoor/Cmdlets/NewAzureRmFrontDoorRoutingRuleObject.cs [hunk 2]
```diff
@@ -126,9 +126,9 @@ public class NewAzureRmFrontDoorRoutingRuleObject : AzureFrontDoorCmdletBase
         public string RedirectProtocol { get; set; }
 
         /// <summary>
-        /// Host to redirect. Leave empty to use use the incoming host as the destination host.
+        /// Host to redirect. Leave empty to use the incoming host as the destination host.
         /// </summary>
```
Comment (markcowl) [inline src/FrontDoor/FrontDoor/Cmdlets/NewAzureRmFrontDoorRoutingRuleObject.cs]: requires an entry in the Frontdoor Changelog

File: src/RecoveryServices/RecoveryServices.SiteRecovery/Properties/Resources.resx [hunk 3]
```diff
@@ -359,10 +359,10 @@ Please provide a storage account with the same location as that of the vault.</v
     <value>Unsupported vault object is passed. Please pass Claasic or ARM vault object.</value>
   </data>
   <data name="ParameterWillBeDeprecatedSoon" xml:space="preserve">
-    <value>Parameter(s) "{0}" will be deprecated soon. Please use use parameter(s) "{1}" instead.</value>
+    <value>Parameter(s) "{0}" will be deprecated soon. Please use parameter(s) "{1}" instead.</value>
```
Comment (markcowl) [inline src/RecoveryServices/RecoveryServices.SiteRecovery/Properties/Resources.resx]: Requires an entry int he SiteRecovery changelog

**Pull request closed without merging**